homeassistant/components/comelit/humidifier.py

```diff
@@ -115,6 +115,7 @@ def __init__(
         self._api = coordinator.api
         self._device = device
         super().__init__(coordinator)
+        self._attr_name = device_class.name.lower()
```

Comment:
This is weird addition? We should name it properly and add a translation key.

Reply:
The bad is that I have 1 name from the bridge for all 3 functions: climate, humidifier, dehumidifier.

In climate.py, with:

```python
    _attr_has_entity_name = True
    _attr_name = None
```

I get **climate.<device_name>**

In humidifier with:

```python
   _attr_has_entity_name = True
   self._attr_name = device_class.name.lower()
```

I get **humidifier.<device_name>_humidifier** and **humidifier.<device_name>_dehumidifier**

Any suggestion is more than welcome

Reply:
I think the suggestion is to use translation keys instead of setting `_attr_name`.

https://developers.home-assistant.io/docs/core/entity#example-of-a-switch-entity-which-is-either-not-the-main-feature-of-a-device-or-is-not-part-of-a-device

Since we already seem to set a translation key and the names in strings.json file seems ok, we can probably just remove this line.

Reply:
```suggestion
```

Reply:
I think I see the problem. We're missing the keys in strings.json file. The existing keys there are under the sensor platform key. Maybe the `humidifier` and `dehumidifier` keys with names there are supposed to belong to the `humidifier` platform?